extensions/2.0/Khronos/KHR_lights_dome/README.md

```diff
@@ -0,0 +1,129 @@
+# KHR_lights_dome
+
+## Contributors
+
+* Ben Houston, Threekit, <https://twitter.com/BenHouston3D>
+* Romain Guy, Google, <https://twitter.com/romainguy>
+
+## Status
+
+Proposed.
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension provides the ability to define a single image-based dome light in a glTF scene.  A dome light illuminate the scene as if the light was coming from a sphere outside of the scene extents.  Only one dome light per scene is supported to avoid confusing this dome light extension with being a specular or diffuse light probe solution.
+
+Many 3D tools and engines support image-based global illumination but the exact technique and data formats employed vary.
+
+### Differences from EXT_lights_image_based
+
+This extension is similar to the EXT_lights_image_based extension but differs in a bunch of important ways.  These differences are motivated by trying to not presuppose the implementation of the rendering method, rather than letting that leak into the data transfer format specification.  It also simplifies what is stored to the minimum, any additional data can be derived as necessary by the rendering engine.
+
+The differences are:
+
+- The cube map is stored in non-prefiltered form (in nits units), rather than prefiltered (in lux units.)
+- Only the top level full resolution cube map is stored, rather than the mip chain.
+- The cube map data, if specified via a PNG, is assumed to be encoded in R11G11B10F, rather than RGBD.
+- No spherical harmonic is stored, if needed it can be generated at run-time.
```

Comment:
I would suggest adding the (diffuse) spherical harmonics, even though it can be generated at run-time we do want to keep runtime processing down and the data to store it is very small.

Reply:
I think if you do run-time pre-filtering you do not need the SH representation at all.  If you only wanted a low resolution light, which an SH offers, you can just give it small 128x128 cube map.  I think forcing implementations to adopt SH is not required for good results, and it increases complexity.